backend/app/controllers/api/v1/investors_controller.rb

```diff
@@ -32,6 +33,12 @@ def fetch_investor
         account = Account.friendly.find(params[:id])
         Investor.find_by!(account_id: account.id)
       end
+
+      def filter_params
+        params.fetch(:filter, {})
```

Comment:
Other endpoints were not using that kind of form, but I like it :+1: . We should also change in other story the rest of endpoints like /locations to have filters in this way `filter[location_type]`

Reply:
Hmm, will try, but I am afraid that FE already uses some of these params 😨

Reply:
something for the future then :)

Reply:
:) in the end, I have changed this with this PR, will see if it blows up something :D

Reply:
oh, but you changed that already? I'd probably separate that in order not to break FE, or maybe check with them how hard it is to change this?